ngnix.conf

```diff
@@ -1,24 +1,8 @@
 server {
     listen 80;
 
-    # declare token is "" (empty str) for original request without args,because $is_args concat any var will be `?`
-    set $token "";
-    # if the request has args update token to "&"
-    # request without args doesn't make sense in this case
-    if ($is_args) {
-        set $token "&";
-    }
-    # file created on container start from env vars (40-create-ghcred.sh)
-    include "/GH_OAUTH_CLIENT.conf";
-    # reverse proxy to github access_token
-    # set rule to /auth explicitly
-    location = /auth {
```

Comment:
Don't we need the GitHub authentication?

Reply:
2e41245472649c4ae228590604a58ce5ae196b08

Reply:
2e41245472649c4ae228590604a58ce5ae196b08

Reply:
Nothing changed here in 2e41245?!